test/suites/upgrade.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+test_upgrade() {
+  reset_systems 4 2 3
+
+  # Perform upgrade test from MicroCloud 1 to 2.
+  if [ "${MICROCLOUD_SNAP_CHANNEL}" = "1/stable" ]; then
+    microceph_target="squid/edge" # TODO: squid/stable when available
+    microovn_target="latest/edge" # TODO: 24.03/stable when available
+    lxd_target="5.21/edge"
```

Comment:
Since we are always testing with `5.21/edge` would you mind updating the check in `test/includes/check.sh` while at it, please?

Reply:
Sure, thanks for the reminder.

Reply:
Fixed